testing/integration/src/builtin.rs

```diff
@@ -16,9 +16,10 @@ use crate::error::Error::{
     FailedToLoadManifest, FailedToSetActor, FailedToSetState, MultipleRootCid, NoCidInManifest,
 };
 
-const BUNDLES: [(NetworkVersion, &[u8]); 2] = [
+const BUNDLES: [(NetworkVersion, &[u8]); 3] = [
     (NetworkVersion::V14, actors_v6::BUNDLE_CAR),
     (NetworkVersion::V15, actors_v7::BUNDLE_CAR),
+    (NetworkVersion::V16, actors_v7::BUNDLE_CAR), // todo bad hack
```

Comment:
?

Reply:
I guess this is because we still don't have a release for v8.

Reply:
Yeah, the integration test runner needs a bundle to load, even if we don't use any actor code. Ideally we could run integration tests without actor bundles - that would also make those tests run much faster.